project_timesheet_time_control/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 Tecnativa - Sergio Teruel
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project timesheet time control",
+    "version": "16.0.1.0.0",
+    "category": "Project",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "maintainers": ["ernestotejeda"],
+    "website": "https://github.com/OCA/project",
+    "depends": [
+        "hr_timesheet_task_domain",
+        "hr_timesheet_task_stage",
```

Comment:
I think we can remove these dependencies as well as just let `hr_timesheet`, as they are not strictly needed.